.eslintrc.json

```diff
@@ -1,5 +1,11 @@
 {
-  "extends": "@podcodar/eslint-config-next",
+  "extends": [
+    "@podcodar/eslint-config-next",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "rules": {
+    "@typescript-eslint/no-non-null-assertion": "off"
```

Comment:
Disabled for Vercel building issues.

Reply:
I think there is abetter way to fix this. Did the non-unused vars failed?

Reply:
https://github.com/typescript-eslint/typescript-eslint/issues/2621#issuecomment-701970389